src/Service/Ecr/src/EcrClient.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+namespace AsyncAws\Ecr;
+
+use AsyncAws\Core\AbstractApi;
+use AsyncAws\Core\Configuration;
+use AsyncAws\Core\Exception\UnsupportedRegion;
+use AsyncAws\Core\RequestContext;
+use AsyncAws\Ecr\Input\GetAuthorizationTokenRequest;
+use AsyncAws\Ecr\Result\GetAuthorizationTokenResponse;
+
+class EcrClient extends AbstractApi
+{
+    /**
+     * Retrieves an authorization token. An authorization token represents your IAM authentication credentials and can be
+     * used to access any Amazon ECR registry that your IAM principal has access to. The authorization token is valid for 12
+     * hours.
+     *
+     * @see https://docs.aws.amazon.com/AmazonECR/latest/APIReference/Welcome.html/API_GetAuthorizationToken.html
```

Comment:
remove `/Welcome.html`. This has been added because of wrong url in manifest.json ;-)

same comment in many places